### ISSUE TITLE
Add mising leetcode problems on readme

### ISSUE THREAD

**Issue by larissalages** [larissalages/code_problems#167]
Not all leet problems that are in the repo are in the readme, please add then and include in the specific section (python/c++/java)

**Comment by ravidyarev** [comment]
Please assign to me

**Comment by Error404m** [comment]
Can I work on this please ..?

**Comment by Deepthi2001** [comment]
can i work on this? or is it already done?

**Comment by gitarthasarma** [comment]
For leetcode problems, it would be better if they are arranged question wise rather than topic wise.
1. The first problem is that many questions are the intersection of different topics so it becomes difficult to categorize, ex: two sum sorted array is both an array and two-pointer problem.
2. The second problem or advantage may be that as questions on the actual leetcode website are arranged question wise, it is better for a reader who visits the repo for a solution, otherwise, they have to go through multiple directories without a result.

**Comment by SamudraThio** [comment]
Agree, with arranging via question numbers rather than topic or language.

**Comment by iamansoni** [comment]
Is this issue still open to working on if yes, You can assign this task to me!
